### DISCUSSION TITLE
Doris无法通过WSL（Oracle Linux7_9）编译，参考官方文档

### DISCUSSION THREAD

**Discussion by wangchong2023** [apache/doris#24705] [RFC]
参考[文档](https://doris.apache.org/zh-CN/docs/dev/install/source-install/compilation-win)

![image](https://github.com/apache/doris/assets/127951479/dcc24366-805b-44df-8616-f3769ef7037c)

已经参考文档安装好了前置软件同时放置好了第三方库
![image](https://github.com/apache/doris/assets/127951479/973bd43d-2b2a-465c-afb8-3f657a036eab)

**Comment by yagagagaga** [comment]
You can [complie with Docker](https://doris.apache.org/zh-CN/docs/dev/install/source-install/compilation-general#%E4%BD%BF%E7%94%A8-docker-%E5%BC%80%E5%8F%91%E9%95%9C%E5%83%8F%E7%BC%96%E8%AF%91%E6%8E%A8%E8%8D%90). It very easy to use.

**Comment by zzzzzzzs** [comment]
也可以尝试用LDB Toolchain 编译
